### PR TITLE
Stopping profile repetition in ScholarX Archive page

### LINKED ISSUE #1575: Remove Profile repetition in ScholarX Archive
**Describe the bug**
Currently there is a profile duplication issue we faced in ScholarX archive mentors page.

**To Reproduce**
1. Go to ScholarX Archive
2. Click on the Mentors section
3. See the error

**Expected behavior**
Remove the profile repetition.

**Additional context**
No.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by RamithaHeshan33**
## Purpose

The purpose of this PR is to fix #1575

## Goals

Stop the profile repetition in [ScholarX Archive](https://sefglobal.org/scholarx/archive/) page

## Approach

oooooooooo

### Screenshots
Before fixing the issue

https://github.com/sef-global/sef-site/assets/142712365/59b8a1ef-e620-487f-a072-b912beb4f806

After fixing the issue

https://github.com/sef-global/sef-site/assets/142712365/a53b2c1b-a500-40ed-9a70-0bee0196c40c
  
### Preview Link

https://pr-1620-sef-site.surge.sh/scholarx/archive/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/archive/functions.js [hunk 1]
```diff
@@ -60,16 +60,22 @@ async function getData() {
...
-        industries.push(data[i].fields)
+        years.push(data[i].year);
+        industries.push(data[i].fields);
         if (data[i].type == "mentor") {
-            mentors.push(data[i])
+            // Check if the mentor's name is already in the Set
+            if (!mentorNames.has(data[i].name)) {
+                mentorNames.add(data[i].name); // Add the mentor's name to the Set
+                mentors.push(data[i]); // Add the mentor to the mentors array
+            }
```
Comment (anjula-sack) [inline scholarx/archive/functions.js]: Good stuff @RamithaHeshan33. Can we make the mentors array a set without the mentorNames?

Comment (RamithaHeshan33) [reply scholarx/archive/functions.js]: > Good stuff @RamithaHeshan33. Can we make the mentors array a set without the mentorNames?

Yes @anjula-sack, I also understood that after submitting the pull request. Sorry about that. We can do the same thing through the mentor's title and mentor's bio. May I use the mentor's title to do this?

Comment (anjula-sack) [reply scholarx/archive/functions.js]: I think the name is best.

Comment (RamithaHeshan33) [reply scholarx/archive/functions.js]: update

How about this?

**Comment by RamithaHeshan33** [comment]
I made some changes based on the screenshot that I sent earlier. Can you check and review it @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Thank you! @RamithaHeshan33

**Pull request merged**